dask/__init__.py

```diff
@@ -8,7 +8,7 @@
 except ImportError:
     pass
 try:
-    from .base import visualize, compute, persist
+    from .base import visualize, compute, persist, thread_state
```

Comment:
If we do keep `thread_state` in `dask`, I at least think it shouldn't be in the top-level. Most dask users have no use for this afaict.

Reply:
That seems fair to me.  I can fix this if this is the direction we decide to go